ui/src/app/pages/workspace/workspace-nav-bar.tsx

```diff
@@ -62,6 +62,7 @@ const styles = reactStyles({
   },
   disabled: {
     color: colors.disabled,
+    cursor: 'not-allowed',
```

Comment:
Disabled nav bar buttons did not have cursor:not-allowed. Adding this helps users better understand that it is disabled and makes our testing more consistent.